IDE/MonoDevelop/MonoDevelop.MonoGame/templates/MonoGameProject.xpt.xml

```diff
@@ -36,9 +36,23 @@
 				<Directory name="Content">
 					<File name="Content.mgcb" src="Common/Content.mgcb" BuildAction="MonoGameContentReference" />
 				</Directory>
+				<Directory name="x64">
+					<File name="libopenal.so.1" src="libs/x64/libopenal.so.1" CopyToOutputDirectory="PreserveNewest" />
```

Comment:
Copy to output is still not enabled, move it into ContentFile tag?

MonoGame.Framework.dll.config is the only file with it enabled.